localstack/utils/crypto.py

```diff
@@ -160,18 +161,22 @@ def unpad(s: bytes) -> bytes:
     return s[0 : -s[-1]]
 
 
-def encrypt(key: bytes, message: bytes, iv: bytes = None) -> bytes:
+def encrypt(key: bytes, message: bytes, iv: bytes = None, aad: bytes = None) -> bytes:
     iv = iv or b"0" * BLOCK_SIZE
-    cipher = Cipher(algorithms.AES(key), modes.CBC(iv))
+    cipher = Cipher(algorithms.AES(key), modes.GCM(iv), backend=default_backend())
```

Comment:
The PR description contains the **what** but misses out on **why** -- could you clarify why this is changed to GCM?